docs/faq.md

```diff
@@ -756,3 +771,48 @@ On examining the container structure with [Dive](https://github.com/wagoodman/di
 The user had used Jib's ability to install extra files into the image ([Maven](https://github.com/GoogleContainerTools/jib/tree/master/jib-maven-plugin#adding-arbitrary-files-to-the-image), [Gradle](https://github.com/GoogleContainerTools/jib/tree/master/jib-gradle-plugin#adding-arbitrary-files-to-the-image)) to install a library file by placing it in `src/main/jib/lib/libfoo.so`. This would normally cause the `libfoo.so` to be installed in the image as `/lib/libfoo.so`. But `/lib` and `/lib64` in the user's base image were symbolic links. Jib does not follow such symbolic links when creating the image. And at container initialization time, Docker treats these symlinks as a file, and thus the symbolic link was replaced with `/lib` as a new directory. As a result, none of the system shared libraries were resolved and dynamically-linked programs failed.
 
 Solution: The user installed the file in a different location.
+
+## Jib CLI 
+ 
+### How does the `jar` command support Standard JARs?
```

Comment:
The FAQ seems more like the technicalities of how a JAR is containerized?
```
### How does the `jar` command containerize Standard JARs?
```

And I think it's worth mentioning that we support both a thin JAR (where other dependencies JARs are specified in `MANIFEST.MF`) and a fat JAR. For thin JARs, adding this link may help: https://docs.oracle.com/javase/tutorial/deployment/jar/downman.html

Reply:
You're right. I cleaned it up a bit to make it less verbose.

Reply:
Oh, actually, I think technically a fat JAR will work, but I realized we need to do more work to better support it. (I believe embedded dependencies do not go into the dependencies layer.) Supporting a fat JAR will be an important feature that we should eventually implement.

For now, I think we should document this limitation (i.e., inefficiency). Basically, I think we can say that we support  thin JARs (with the link in the comment above), and then mention the current limitation and that we are working on better fat JAR support. (Also this FAQ should probably have a link to the other FAQ for the Spring Boot fat jar case.) Thoughts?